api/app/controllers/review.controller.ts

```diff
@@ -161,7 +184,7 @@ export async function findOne(req: Request, res: Response) {
           message: `Review with id=${id} does not exist`,
         });
       }
-      res.status(200).send(data);
+      return res.status(200).send(transcriptWrapper(data.transcript))
```

Comment:
`transcriptWrapper` is async, this must be:
```suggestion
      return res.status(200).send(await transcriptWrapper(data.transcript))
```